docs/source/glossary.rst

```diff
@@ -3,74 +3,88 @@ Glossary
 
 .. glossary::
 
-  Node
-    Machine running all the components necessary for a working blockchain
-    (including the Validator, the REST API, at least one transaction processor,
-    and the PBFT algorithm itself). In this RFC, unless otherwise specified,
-    it can be assumed that *Node* refers to the PBFT component of the machine.
-
-  Server
-    Synonym for node.
-
-  Replica
-    Synonym for node.
-
-  Validator
-    Component of a node responsible for interactions with the blockchain.
-    Interactions with the validator are abstracted by the Consensus API.
-
   Block
-    A part of the
-    `blockchain <https://en.wikipedia.org/wiki/Block chain>`__,
-    containing some operations and a link to the previous block.
-
-  Primary
-    Node in charge of making the final consensus decisions and committing to
-    the blockchain.  Additionally is responsible for publishing the blocks given
-    to it by the Consensus API, and starting the consensus process.
-
-  Secondaries
-    Auxiliary nodes used for consensus.
+    Part of the `blockchain <https://en.wikipedia.org/wiki/Block chain>`__
+    that contains one or more operations (called `transactions`) and a link to
+    the previous block.
 
   Client
     Machine that sends requests to and receives replies from the network of
-    nodes. PBFT has no direct interaction with clients; the Validator bundles
-    all client requests into blocks and sends them through the Consensus API to
-    the consensus algorithm.
+    Sawtooth nodes. PBFT has no direct interaction with clients; the validator
+    bundles all client requests into blocks and sends them through the
+    consensus API to the consensus engine.
 
-  Consensus seal
-    Proof that a block underwent consensus.
+  Consensus API
+    Sawtooth component that abstracts consensus-related interactions between
+    the validator and a consensus engine. The consensus API allows `dynamic
+    consensus`, a feature that allows a choice of consensus for a Sawtooth
+    network.
 
-  Block duration
-    How many seconds to wait before trying to publish a block.
+  Consensus engine
+    Component that provides consensus functionality for a Sawtooth
+    network. The consensus engine communicates with the validator through
+    the consensus API.
+
+  Consensus seal
+    Proof that a block went through consensus.
 
   Member node
     Sawtooth node that participates in PBFT consensus. Membership is controlled
     by the on-chain setting ``sawtooth.consensus.pbft.members``.
 
   Message
-    Block, with additional information (see :ref:`pbft-arch-message-types`).
-
-  Working block
-    The block that has been initialized but not finalized, and is currently
-    being committed to.
-
-  View
-    The period of time of PBFT when the current primary is in charge. The view
-    changes when the primary is deemed faulty (see
-    :ref:`view-changes-choosing-primary-label`).
+    Block with additional consensus-related information (see
```

Comment:
I don't think a Message is a Block

Reply:
Hmmm.  That was in the original document (I didn't change that word.)  I'll check with @ltseeley.

Reply:
We decided on this wording: 

Message
Consensus-related information that nodes send to one another. For more information, see [Consensus Messages](https://sawtooth.hyperledger.org/docs/pbft/nightly/master/architecture.html#consensus-messages).